src/providers/github/index.ts

```diff
@@ -66,6 +72,40 @@ export async function fetchGitHubSponsors(token: string, login: string, type: st
       createdAt: raw.createdAt,
     }))
 
+  try {
```

Comment:
We should provide an option to opt-out of past sponsors.

Reply:
If the user provides tiers that include `monthlyDollars` that are less than zero or equal, we will fetch the past sponsors.